cmd/delete/matchable_task_resource_attribute.go

```diff
@@ -0,0 +1,98 @@
+package delete
+
+import (
+	"context"
+
+	"github.com/flyteorg/flytectl/cmd/config"
+	"github.com/flyteorg/flytectl/cmd/config/subcommand"
+	cmdCore "github.com/flyteorg/flytectl/cmd/core"
+	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/admin"
+	"github.com/flyteorg/flytestdlib/logger"
+)
+
+const (
+	taskResourceAttributesShort = "Deletes matchable resources of task attributes"
+	taskResourceAttributesLong  = `
+Deletes task  resource attributes for given project,domain combination or additionally with workflow name.
+
+Deletes task resource attribute for project and domain
+Here the command delete task resource attributes for  project flytectldemo and development domain.
+::
+
+ flytectl delete task-resource-attribute -p flytectldemo -d development 
+
+
+Deleting task resource attribute using config file
+Here the command deletes task resource attributes from the config file tra.yaml
+eg:  content of tra.yaml which will use the project domain and workflow name for deleting the resource
+
+::
+
+ flytectl delete task-resource-attribute --attrFile tra.yaml
+
+
+.. code-block:: yaml
+
+	Domain: development
+	Project: flytectldemo
+	Workflow: ""
+	defaults:
```

Comment:
why are there specified defaults & limits for delete? shouldn't the identifier (domain, project and optionally workflow) be all that is required to delete?

Reply:
Yes it requires only the domain,project or workflow optionally. For delete it only uses those fields to call the corr. admin API.
Wanted to use the same file for update and delete .
Even in kubectl when we call 
kubectl delete -f  <config-file> 
It ignores the details but uses the identifiers to delete the resources. Have tried to use the same semantics here so that it becomes easier for the user .

Also there is another option which is supported by explicitly specifying all these through commandline
```
flytectl delete task-resource-attribute -p flytectldemo -d development 
 flytectl delete task-resource-attribute -p flytectldemo -d development core.control_flow.run_merge_sort.merge_sort
```